benchmarks/models.py

```diff
@@ -0,0 +1,77 @@
+import math
```

Comment:
same, header

Reply:
the filename models.py seems to be too generic. Can you rename to it something more specific or do we want to put all model related classes in a single file?

Reply:
Added a header.

I want to add a models and data folder where we can add specific models and datasets sort of like a mxn combination. This is happening in a followup PR where I will allow the benchmark to specify different models and datasets to run with the same training loop.

Reply:
sounds good. rename later is totally fine.